src/test/kotlin/com/sourcegraph/cody/autocomplete/action/TriggerAutocompleteActionTest.kt

```diff
@@ -0,0 +1,25 @@
+package com.sourcegraph.cody.autocomplete.action
+
+import com.intellij.testFramework.fixtures.BasePlatformTestCase
+import com.sourcegraph.cody.autocomplete.CodyAutocompleteManager
+
+class TriggerAutocompleteActionTest : BasePlatformTestCase() {
+
+  fun `test autocomplete`() {
+    // TODO: replace this condition with test framework-level feature (maybe
+    // something similar to SkipException from TestNG). Tests should be "yellow"
+    // (but still passing), not "green".
+    if (missingToken()) return
```

Comment:
There is cool SkipException feature in TestNG. We can throw that during test and it will be marked as skipped. Intellij supports that very well and test are yellow, but they are not marked as yellow. I'm looking for something similar to this in JUnit 3. We can bump version, but this causes other issues.

Another option is to add this test to different source set and create separate task for integration tests like this. Something like `./gradlew testIntegration`.

Reply:
`./gradlew testIntegration` sounds great and it seem to be a common practice